server/database/db_settings.py

```diff
@@ -0,0 +1,27 @@
+import os
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+load_dotenv()
+
+POSTGRES_DB_NAME = os.getenv("POSTGRES_DB_NAME") or "postgres"
+POSTGRES_USERNAME = os.getenv("POSTGRES_USERNAME") or "postgres"
+POSTGRES_HOST = os.getenv("POSTGRES_HOST") or "localhost"
+POSTGRES_PORT = os.getenv("POSTGRES_PORT") or "5432"
+POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD") or "admin"
```

Comment:
Wouldn't it be better to have these default values in a default `.env` too?
Or, in my opinion, it would be great to have a `.env` with all defaults and if the value isn't specified in `.env` throw an error. It would make the process of database configuration easier and more transparent

Reply:
Fixed